app/org/maproulette/controllers/api/ChallengeController.scala

```diff
@@ -755,8 +755,8 @@ class ChallengeController @Inject() (
           challenge.extra.exportableProperties match {
             case Some(ex) =>
               if (!ex.isEmpty) {
-                var propsToExport = ex.replaceAll("\\s", "").split(",")
-                propsToExport.foreach(pe => propsToExportHeaders += pe)
+                var propsToExport = ex.split(",")
+                propsToExport.foreach(pe => propsToExportHeaders += pe.trim)
```

Comment:
Trim will only remove whitespace characters at the beginning and end of the string. You are changing it such that this will only happen for each element found separated by the comma. I am guessing that most likely this is what was originally intended, are you sure there are no cases that required whitespaces to be removed within the found elements?

Reply:
It seems that this is to support allowing whitespace in the exported column headers. That is what I was missing. If that is the case, then it looks good.